src/read/read_ref.rs

```diff
@@ -70,18 +65,7 @@ where
         K: Borrow<Q>,
         Q: Hash + Eq,
     {
-        let inner = self.guard.as_ref()?;
-        if !inner.is_ready() {
-            return None;
-        }
-        inner.data.get(key).map(Values::user_friendly)
-    }
-
-    /// Returns true if the writer has destroyed this map.
-    ///
-    /// See [`WriteHandle::destroy`].
-    pub fn is_destroyed(&self) -> bool {
-        self.guard.is_none()
+        self.guard.data.get(key).map(Values::user_friendly)
```

Comment:
Maybe I'm missing something, but I don't think the check for whether the map is ready now gets performed anywhere? It's not to my knowledge applied inside `handle` at least?